meta/saisanitycheck.c

```diff
@@ -623,6 +623,7 @@ void check_attr_object_type_provided(
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_IPV4:
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_IPV6:
         case SAI_ATTR_VALUE_TYPE_ACL_ACTION_DATA_IP_ADDRESS:
+        case SAI_ATTR_VALUE_TYPE_IPV4:
```

Comment:
move it to line 602, this set only contains ACL_ACTION entries

Reply:
Kamil, This is caused by accidental merge